8.0 [FIX] Modified ir.rule to allow read document page

luiseevaquer opened the pull request:
Issue#4557

Commit:
[FIX] document_page_approval_levels: Modified ACL to allow read document page

sabrinagnu commented:
@lescobarvx once I reviewed deeply this change, first of all, the title of this PR is wrong, as your are not changing a ACL, you are changing an ir.rule, second the real fix for this issue is create this new ir.rule:

![record rules odoo](https://user-images.githubusercontent.com/7597456/29694941-4f7291e4-8905-11e7-9526-e3aba66ee128.png)

Because what is really missing is to allow to and user with a profile bigger than "user" to see and change (not delete) documents from all users, with the current change on this PR we are allowing to every users to see all documents and this is not the expected behaviour, what is expected is:
User: can se, and everything over their own documents.
Approver or manager: can see and edit documents from every users.

The documents can be deleted only by the user that created the document itself.

May you please set on this PR the new ir.rule?.

Best regards!

sabrinagnu commented on document_page_approval_levels/security/res_groups.xml:
```diff
@@ -18,7 +18,7 @@
             <field name="groups" eval="[(4, ref('base.group_document_user'))]" />
-            <field name="perm_read" eval="1"/>
+            <field name="perm_read" eval="0"/>
             <field name="perm_write" eval="1"/>
```
New ir.rule must be added.

Commit:
[FIX] document_page_approval_levels: Added and modified ir.rule for:

    - Allow see and everything over their documents.
    - Allow see and edit documents if user is approver or manager.
    - Allow only see if a single user.

Commit:
[FIX] document_page_approval_levels: Changed and removed unnecessary rule

Commit:
[FIX] document_page_approval_levels: Added rule to allow see documents
of other users
